mocks/medl-mock/src/main/java/no/nav/medl2/rest/api/v1/MedlemskapsunntakMock.java

```diff
@@ -44,7 +46,7 @@ public class MedlemskapsunntakMock {
 
     @SuppressWarnings("unused")
     @GET
-    @Path("/{unntakId}")
+    @Path("/medlemskapsunntak/{unntakId}")
```

Comment:
Er du sikker på at du vil legge inn en ekstra _medlemskapsunntak/_ her? Mulig k9 bruker denne også.

Reply:
Aner ikke om noen bruker denne for å slå opp enkeltunntak. vi gjør det ikke.